Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -1294,6 +1294,8 @@ public void RaiseSpeechCommandRecognized(IMixedRealityInputSource source, Recogn
             // Create input event
             speechEventData.Initialize(source, confidence, phraseDuration, phraseStartTime, command);
 
+            FocusProvider?.OnSpeechKeywordRecognized(speechEventData);
```

Comment:
>FocusProvider?.OnSpeechKeywordRecognized(speechEventData); [](start = 12, length = 58)

It would be cleaner if our FocusProvider implementation received speech events registering as a global listener instead.

Reply:
Will also suggested this, I didn't do this because I wanted to follow the existing pattern. But it looks like the existing pattern is incorrectly checking null anyway I'll make the change

Reply:
Okay, I see why it was not created as a global listener. The FocusProvider is actually not a gameObject, so cannot be registered as a global listener using `MixedRealityToolkit.InputSystem.Register(gameObject);`. Will keep it as is since I don't know of a solution to register the MRTK input system with a class that is not a game object.